.stylelintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this file should be taken out of this PR – it should already be in `master` I think.